server.ts

```diff
@@ -0,0 +1,45 @@
+const [, major] = process.version.match(/v(\d+)\.(\d+)\.(\d+)/);
+if (+major < 5) {
+  console.log("\x1b[31mWARNING: YOU SHOULD BE RUNNING THIS WITH NODE >= 5!! FOUND VERSION:", process.version, "\x1b[0m");
+  process.exit(1);
+}
+
+
+var bee    = require('beeline');
```

Comment:
does anyone else use this? should we stick to a known router?

might be important e.g. when using standardized request middleware, swagger, etc